client/src/Log/components/LogList.tsx

```diff
@@ -0,0 +1,76 @@
+import { FC, useState } from 'react'
+import { useQuery } from '@apollo/client'
+
+// common
+import Spinner from 'common/components/Spinner'
+import ErrorFallback from 'common/components/ErrorFallback'
+import SearchBar from 'common/components/SearchBar'
+import Pagination from 'common/components/Pagination'
+import { numberWithCommas } from 'common/helpers'
+
+// log
+import { QUERY_LOG_CONNECTION_LIST } from 'Log/query'
+import LogTable from './LogTable'
+
+const LogList: FC = () => {
+  const [currentPage, setCurrentPage] = useState(1)
+  const [lastCursor, setLastCursor] = useState(undefined)
+  const PAGE_SIZE = 10
+
+  const {
+    data: connectionData,
```

Comment:
It seems there is no need to assign new names, can just use original: `data`, `error`...